omod/src/main/webapp/pages/editSection.gsp

```diff
@@ -215,22 +215,13 @@ ${ ui.includeFragment("uicommons", "validationMessages")}
             <input id="patientUuid" type="hidden" name="patientUuid" value="${patientUuid}"/>
         </section>
 
-        <div id="confirmation">
-            <span class="title">${ui.message("registrationapp.patient.confirm.label")}</span>
-            <div class="before-dataCanvas"></div>
-            <div id="dataCanvas"></div>
-            <div class="after-data-canvas"></div>
-            <div id="confirmationQuestion">
-                ${ui.message("registrationapp.confirm")}
+       <div id="confirmationQuestion">
```

Comment:
@jwamalwa  what did you change here, it seems this there is no change here thanks for this work again